.github/workflows/release.yaml

```diff
@@ -31,13 +32,13 @@ jobs:
       - name: Setup dotnet
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: "5.0.x"
+          dotnet-version: "6.0.x"
```

Comment:
did we update target frameworks in msbuild file ?

https://github.com/GetStream/stream-chat-net/blob/master/src/stream-chat-net/stream-chat-net.csproj#L7

Reply:
yes. 6.0 is included there.

Reply:
there's no EOL stuff, or any breaking changes as far as I know

Reply:
👍🏻